server/adaptors/integrations/__data__/repository/aws_cloudtrail/schemas/aws_cloudtrail-1.0.0.mapping.json

```diff
@@ -0,0 +1,86 @@
+{
+  "template": {
+    "mappings": {
+      "_meta": {
+        "version": "1.0.0",
+        "catalog": "observability",
+        "type": "logs",
+        "component": "cloudtrail"
+      },
+      "properties": {
+        "aws": {
+          "type": "object",
+          "properties": {
+            "cloudtrail": {
```

Comment:
Diff reports lots of fields present in the data but not in the schema here:

```diff
- aws.cloudtrail.additionalEventData: {"SignatureVersion": "SigV4", "CipherSuite": "ECDHE-RSA-AES128-GCM-SHA256", "bytesTransferredIn": 0, "AuthenticationMethod": "AuthHeader", "x-amz-id-2": "dNg98Ncsm0R/PygSHq0b0Qiyx2TDUq9jgC9wSxasWFiQ/hJPz1rv2RWiUWErUcWAZyJqKoxckNM=", "bytesTransferredOut": 544}
- aws.cloudtrail.eventID: "8e6ffb2a-af42-47e6-9da9-8d90adbdc937"
- aws.cloudtrail.managementEvent: true
- aws.cloudtrail.readOnly: true
- aws.cloudtrail.recipientAccountId: "347283850106"
- aws.cloudtrail.requestID: "8RJVMXE049D1QTJ1"
- aws.cloudtrail.requestParameters: {"bucketName": "aws-cloudtrail-mytrail-347283850106-059ac1d5", "Host": "aws-cloudtrail-mytrail-347283850106-059ac1d5.s3.eu-west-1.amazonaws.com", "acl": ""}
- aws.cloudtrail.resources: [{"accountId": "347283850106", "type": "AWS::S3::Bucket", "ARN": "arn:aws:s3:::aws-cloudtrail-mytrail-347283850106-059ac1d5"}]
- aws.cloudtrail.sharedEventID: "535103f5-cc74-4812-90dd-6861bfa00ff3"
- aws.cloudtrail.userAgent: "cloudtrail.amazonaws.com"
- aws.cloudtrail.userIdentity.invokedBy: "cloudtrail.amazonaws.com"
- aws.cloudtrail.userIdentity.type: "AWSService"
```

Are we okay with all of these being dropped during ingestion?